### ISSUE TITLE
 'MobileAppendix' could not be found

### ISSUE THREAD

**Issue by ofir-b-HDEG** [Unity-Technologies/BuildReportInspector#32]
I am getting this error when after adding the script into my project:
Assets\Scripts\Editor\BuildReportInspector.cs(45,17): error CS0246: The type or namespace name 'MobileAppendix' could not be found (are you missing a using directive or an assembly reference?)

Running on Unity 2022.3.30f1 on Windows 11. Target build platform at the moment - Android.

**Comment by Xeonzinc** [comment]
I think the instructions are misleading - I believe you need to download all files / folders in the editor folder for it to work

**Comment by SkowronskiAndrew** [comment]
Yes @xeonzinc.  I agree it was the instructions that were problematic. I think originally the entire Inspector was a single file so you didn't technically need the full package.  But additional source files are needed so installing the entire package by pointing to the local clone or downloaded code is the only way that is recommended.  I've remove the comment saying that it could be installed with one file.
